cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -787,6 +787,19 @@ rules:
   - csidrivers
   verbs:
   - create
+- apiGroups:
+    - "admissionregistration.k8s.io"
+    - ""
```

Comment:
Why is this required?

Reply:
This is done to add necessary privileges in rook service account to create and update resources from admissionregistration.k8s.io